### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by mshahulpm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+## This is Demo project for demos need to implement some othwr tools for better way to do it
```
Comment (shahulpm) [inline README.md:1]: Add multiple oprion

**Review by shahulpm — COMMENTED** [review]
Need to implement requested changes

**Comment by mshahulpm** [comment]
sure

**Review by shahulpm — APPROVED** [review]
Looking good mate

**Pull request merged**